cms/grading/__init__.py

```diff
@@ -347,6 +350,24 @@ def format_status_text(status, translator=None):
                      "text: %r", status, exc_info=True)
         return translator("N/A")
 
+def AIOTranslator(translator):
+    def simplify_status_text(status):
+        mapping_startswith = [
+            ("Execution failed because of sandbox error",              "Judge error, please notify judges"),
+            ("Execution killed because of forbidden syscall",          "Program killed due to illegal operation. Please check the rules regarding allowable system calls"),
+            ("Execution timed out (wall clock limit exceeded)",        "Time limit exceeded"),
+            ("Execution timed out",                                    "Time limit exceeded"),
+            ("Execution killed because of forbidden file access:",     "Forbidden file access. Please check your input/output filenames match those in the problem"),
+            ("Execution killed with signal 11",                        "Program crashed after accessing or requesting invalid memory"),
```

Comment:
Could you dupe this for signal 10 too? (SIGSEGV is 11, SIGBUS is 10; it's somewhat architecture/machine/specific line-of-code dependent which one is thrown but they almost always mean the same thing.)